sycl/source/detail/kernel_compiler/kernel_compiler_opencl.cpp

```diff
@@ -98,7 +98,7 @@ void loadOclocLibrary(const std::vector<uint32_t> &IPVersionVec) {
     }
     return true;
   };
-  for (auto result : OclocPaths) {
+  for (const auto &result : OclocPaths) {
```

Comment:
IIUC, `std::string_view` of a string literal is well-defined. Can you change line 74 to use them (instead of `std::string`) and spellout explicit `for (std::string_view OclocPath : OclocPaths)` here?

Reply:
I switched to `std::string_view`  for the declaration. But won't Coverity still tag it if it's not const ref?

Reply:
Not sure, but I don't think it should. `std::string_view` is a small type that is intended to be passed by value.